client/flutter/lib/api/content_loading.dart

```diff
@@ -6,12 +6,12 @@ import 'package:flutter/material.dart';
 import 'content_bundle.dart';
 
 class ContentLoading {
-  static final networkLoadingEnabled = false;
+  static final networkLoadingEnabled = true;
   static final ContentLoading _singleton = ContentLoading._internal();
 
   // TODO: Real URL
```

Comment:
Can this `TODO` now be removed? 🙂

Reply:
No, won't remove until confirmed with Bob